docs/deployment/config_descriptions.md

```diff
@@ -64,14 +119,18 @@ These values will be imported via kes in your configuration file.   You should e
 ```yaml
 cmr:
   username: '{{CMR_USERNAME}}'
-  provider: CUMULUS
-  clientId: '<replace-with-daac-name>-{{stackName}}'
+  provider: myCmrProvider
```

Comment:
```suggestion
  provider: <replace-with-cmr-provider>
```
Up to you - I propose it because the other "replace this value" in `clientId` uses this kind of structure.

Reply:
👍  6c267ec